content/teknologi/altinnstudio/altinn-api/app-api/data-element/_index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Data elements
+description: The App API to work with the data elements associated with an instance
+toc: true
+tags: [api]
+weight: 100
+---
+
+## Overview
+
+A Data element consist of two parts; its metadata document and the actual data blob. This API work primarily with the blob while keeping the metadata document updated.
+
+**basePath**
+```http
+{org}/{appname}/instances/{instanceOwnerPartyId}/{instanceGuid}/data
+```
+
+## Get data 
+
+Endpoint for downloading the data blob.
+
+```http
+GET basePath/{dataGuid}
+Accept: application/{xml/json}
+```
+
+The response will depend on the type of blob. There are currently 2 primary types; forms and attachments.
```

Comment:
I would like to see the terms here be repeated in the sub section headers.. e.g. forms -> Response with form or form data -> respose with form data.